### PR TITLE
Added formatDate function and fix film's card view

### PR DESCRIPTION
**Pull request by AntonBurchak**
Task: https://trello.com/c/U3ZdHWLz/44-6-fix-films-design
Commit: https://github.com/JIeCoRyI3/the-movie-db/pull/36/commits/bc5d323af893a9f7df8a723aa95b48890d711262

### CODE REVIEW

File: src/components/FilmCardComponent/FilmCard.jsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { withRouter } from 'react-router';
 import PropTypes from 'prop-types';
 import styles from './FilmCard.module.css';
 import ApiClient from '../../api/apiClient';
+import { formatDate } from '../../utils/utils';
```
Comment (LordGemma) [inline src/components/FilmCardComponent/FilmCard.jsx:6]: Please check this path. Are you sure that this file exists?

File: src/utils/utils.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { formatDate } from './utils';
+
+describe('formatDate()', () => {
+    it('Should return correct date', () => {
+        expect(formatDate('2020-12-20')).toBe('20.12.2020');
+    });
+
+    it('Should return error', () => {
+        expect(() => formatDate('2020-20-202')).toThrowError(Error);
```
Comment (LordGemma) [inline src/utils/utils.test.js:9]: Could you explain the purpose of this test?

File: src/components/FilmCardComponent/FilmCard.jsx (hunk 1 above)
Comment (AntonBurchak) [reply src/components/FilmCardComponent/FilmCard.jsx:6]: Of course, I've added it

File: src/utils/utils.test.js (hunk 2 above)
Comment (AntonBurchak) [reply src/utils/utils.test.js:9]: This test indicate that formatDate function should throw error when accept incorrect date string

File: src/components/FilmCardComponent/FilmCard.jsx (hunk 1 above)
Comment (AntonBurchak) [reply src/components/FilmCardComponent/FilmCard.jsx:6]: Yeah, I'm forgot to add this file to commit. I've updated.

**Pull request merged**